src/languages/verilog.js

```diff
@@ -6,67 +6,490 @@ Description: Verilog is a hardware description language used in electronic desig
+      "$onehot",
+      "$isunknown",
+      "$fatal",
+      "$warning",
+      "$dimensions",
+      "$right",
+      "$high",
+      "$size",
+      "$asin",
+      "$acos",
+      "$atan",
+      "$atan2",
+      "$hypot",
+      "$sinh",
+      "$cosh",
+      "$tanh",
+      "$asinh",
+      "$acosh",
+      "$atanh",
+      "$countones",
+      "$onehot0",
+      "$error",
+      "$info",
+      "$random",
+      "$dist_chi_square",
+      "$dist_erlang",
+      "$dist_exponential",
+      "$dist_normal",
+      "$dist_poisson",
+      "$dist_t",
+      "$dist_uniform",
+      "$q_initialize",
+      "$q_remove",
+      "$q_exam",
+      "$async$and$array",
+      "$async$nand$array",
+      "$async$or$array",
+      "$async$nor$array",
+      "$sync$and$array",
+      "$sync$nand$array",
+      "$sync$or$array",
+      "$sync$nor$array",
+      "$q_add",
+      "$q_full",
+      "$psprintf",
+      "$async$and$plane",
+      "$async$nand$plane",
+      "$async$or$plane",
+      "$async$nor$plane",
+      "$sync$and$plane",
+      "$sync$nand$plane",
+      "$sync$or$plane",
+      "$sync$nor$plane",
+      "$system",
+      "$display",
+      "$displayb",
+      "$displayh",
+      "$displayo",
+      "$strobe",
+      "$strobeb",
+      "$strobeh",
+      "$strobeo",
+      "$write",
+      "$readmemb",
+      "$readmemh",
+      "$writememh",
+      "$value$plusargs",
+      "$dumpvars",
+      "$dumpon",
+      "$dumplimit",
+      "$dumpports",
+      "$dumpportson",
+      "$dumpportslimit",
+      "$writeb",
+      "$writeh",
+      "$writeo",
+      "$monitor",
+      "$monitorb",
+      "$monitorh",
+      "$monitoro",
+      "$writememb",
+      "$dumpfile",
+      "$dumpoff",
+      "$dumpall",
+      "$dumpflush",
+      "$dumpportsoff",
+      "$dumpportsall",
+      "$dumpportsflush",
+      "$fclose",
+      "$fdisplay",
+      "$fdisplayb",
+      "$fdisplayh",
+      "$fdisplayo",
+      "$fstrobe",
+      "$fstrobeb",
+      "$fstrobeh",
+      "$fstrobeo",
+      "$swrite",
+      "$swriteb",
+      "$swriteh",
+      "$swriteo",
+      "$fscanf",
+      "$fread",
+      "$fseek",
+      "$fflush",
+      "$feof",
+      "$fopen",
+      "$fwrite",
+      "$fwriteb",
+      "$fwriteh",
+      "$fwriteo",
+      "$fmonitor",
+      "$fmonitorb",
+      "$fmonitorh",
+      "$fmonitoro",
+      "$sformat",
+      "$sformatf",
+      "$fgetc",
+      "$ungetc",
+      "$fgets",
+      "$sscanf",
+      "$rewind",
+      "$ftell",
+      "$ferror"
+    ]
   };
+  const BUILT_IN_CONSTANTS = [
+    "__FILE__",
+    "__LINE__"
+  ];
+  const DIRECTIVES = [
+    "define",
+    "begin_keywords",
+    "celldefine",
+    "default_nettype",
+    "define",
+    "else",
+    "elsif",
+    "end_keywords",
+    "endcelldefine",
+    "endif",
+    "ifdef",
+    "ifndef",
+    "include",
+    "line",
+    "nounconnected_drive",
+    "pragma",
+    "resetall",
+    "timescale",
+    "unconnected_drive",
+    "undef",
+    "undefineall"
```

Comment:
```suggestion
    "begin_keywords",
    "celldefine",
    "default_nettype",
    "default_decay_time",
    "default_trireg_strength",
    "define",
    "delay_mode_distributed",
    "delay_mode_path",
    "delay_mode_unit",
    "delay_mode_zero",
    "else",
    "elsif",
    "end_keywords",
    "endcelldefine",
    "endif",
    "ifdef",
    "ifndef",
    "include",
    "line",
    "nounconnected_drive",
    "pragma",
    "resetall",
    "timescale",
    "unconnected_drive",
    "undef",
    "undefineall"
```

Added some missing directives according to [UMBC](https://www.csee.umbc.edu/portal/help/VHDL/verilog/compiler.html) and removed a duplicate.